app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -7,6 +7,10 @@ class RunHostJob < Actions::EntryAction
       middleware.do_not_use Dynflow::Middleware::Common::Transaction
       middleware.use Actions::Middleware::HideSecrets
 
+      def queue
+        :remote_execution
```

Comment:
How about replacing all the occurences of `:remote_execution` with a constant?

Reply:
Good idea, updated.